frontend/src/pages/home/HomeHint.tsx

```diff
@@ -4,28 +4,28 @@ import {
   Card,
   CardBody,
   CardHeader,
+  Content,
   Flex,
   FlexItem,
   PageSection,
-  Content,
 } from '@patternfly/react-core';
 import { TimesIcon } from '@patternfly/react-icons';
 import { useBrowserStorage } from '~/components/browserStorage';
-import TitleWithIcon from '~/concepts/design/TitleWithIcon';
-import { ProjectObjectType } from '~/concepts/design/utils';
-import ExternalLink from '~/components/ExternalLink';
-import { SupportedArea, useIsAreaAvailable } from '~/concepts/areas';
 
-const HomeHint: React.FC = () => {
+type HomeHintProps = {
+  title: React.ReactNode;
+  body: React.ReactNode;
+  isDisplayed: boolean;
+  image?: React.ReactNode;
+};
+
+const HomeHint: React.FC<HomeHintProps> = ({ title, image, body, isDisplayed }) => {
   const [hintHidden, setHintHidden] = useBrowserStorage<boolean>(
     'odh.dashboard.landing.hint',
     false,
   );
```

Comment:
this key needs to chnage based on the type of home hint. becuase if we change the home hint then we will want users that have hidden the hint before to see the new one. probably passing some key or id in shoudl do the trick and slap it on the end of the key